### ISSUE TITLE
Fix react v0.14.0-rc1 warning

### ISSUE THREAD

**Issue by cesarandreu** [nfl/react-helmet#42]
```
Warning: SideEffect(Helmet)(...): React component classes must extend React.Component.
```

It's caused by react-side-effect, so once they release an update it should only require updating the version that react-helmet uses.

Associated issue: https://github.com/gaearon/react-side-effect/issues/16

**Comment by cwelch5** [comment]
Thanks. I'll keep an eye out.

**Comment by xpepermint** [comment]
+1

**Comment by gaearon** [comment]
This is fixed in `react-side-effect@1.0.2` via https://github.com/gaearon/react-side-effect/pull/14.
